tests/bootstrap.php

```diff
@@ -50,5 +51,5 @@
 
     $kernel->shutdown();
 
-    $fs->remove(__DIR__.'/../var');
+    Configuration::shutdown();
```

Comment:
This was needed otherwise in migrate mode, Foundry stays "booted" and did not throw `NotBootedException` in `PersistentProxyObjectFactory`